questionary/prompts/checkbox.py

```diff
@@ -38,6 +38,7 @@ def checkbox(
     use_jk_keys: bool = True,
     use_emacs_keys: bool = True,
     instruction: Optional[str] = None,
+    show_description: bool = True,
```

Comment:
@viniciusdc I've changed this to `True` by default, because that is what the user expects when they set a description on a `Choice`. Plus, since there are no descriptions yet, previous code is not affected by this.